redaxo/src/core/lib/sql/sql.php

```diff
@@ -1928,6 +1913,23 @@ public static function checkDbConnection($host, $login, $password, $database, $c
         return $err_msg;
     }
 
+    /**
+     * @param mixed $value
+     * @param string|array|callable|null $type
+     */
+    private function castValue($value, $type)
+    {
+        if (null === $type) {
+            return $value;
+        }
+
+        if (is_array($type) || 'array' === $type || is_string($type) && str_starts_with($type, 'array[')) {
+            $value = json_decode($value, true);
+        }
```

Comment:
hmm.. ich versteh nicht wann/warum der $type hier json sein kann?
das haben wir bei rex_get/post/.. nicht, oder?

Reply:
Arrays speichern wir in der DB als JSON.
bei rex_get etc sind es bereits Arrays.

Reply:
ohh jetzt versteh ich es... das if geht auf den $type und im if arbeiten wir auf $value